app/components/login.jsx

```diff
@@ -1,202 +1,318 @@
-
-  render () {
-    let classes = [ 'syn-login' ];
-
-    if ( this.props.show ) {
-      classes.push('syn--visible');
-    }
-
-    let content = (
-      <div onClick={this.stopPropagation.bind(this)}>
-      {(this.socialMeida)?(
-        <ButtonGroup block>
-          <Button medium primary onClick={ this.loginWithFacebook } className="login-with-facebook" ref='facebook'>
-            <Icon icon="facebook" />
-            <span className={ Component.classList(this) }> Facebook</span>
-          </Button>
-
-          <Button medium info onClick={ this.loginWithTwitter } className="login-with-twitter">
-            <Icon icon="twitter" />
-            <span> Twitter</span>
-          </Button>
-        </ButtonGroup>
-        ):null}
-
-        <div className="syn-form-group">
-          <label>Email</label>
-          <Input type='email' block autoFocus required medium placeholder="Email" ref="email" name="email" />
-        </div>
-
-        <div className="syn-form-group">
-          <label>Password</label>
-          <Input type='password' block required placeholder="Password" ref="password" medium name="password" />
-        </div>
-
-        <div className="syn-form-group syn-form-submit">
-          <Submit block large success radius>Login</Submit>
-        </div>
-
-        <Row>
-          <Column span="50" gutter className="text-left">
-            <p style={{margin: 0}}>Not yet a user?</p><a href="#" onClick={ this.signUp.bind(this) } ref='signup'>Sign up</a>
-          </Column>
-
-          <Column span="50" text-right gutter className="forgot-password">
-            <p style={{margin: 0}} className="forgot-password-label">Forgot password?</p>
-            <a
-              href              =   "#"
-              className         =   "forgot-password-link"
-              onClick           = { this.forgotPassword }
-            >
-            Click here
-            </a>
-          </Column>
-        </Row>
-      </div>
-    );
-
-    if ( this.state.info ) {
-      content = ( <Loading message="Loggin you in..." /> );
-    }
-
-    else if ( this.state.successMessage ) {
-      content = ( <div></div> );
+// https://github.com/EnCiv/civil-pursuit/issues/150
+
+// CHANGELOG
+// 7/11/24: removed instances of useAuth to try to get interface working, will fix later
+// 7/12/24: removed "ref" from login.jsx file since its deprecated, trying to get some stories up
+// 7/13/24: changed file name to login.jsx after installing civil-client
+// 7/15/24: i had to change stuff on webpack-dev to get the email updating properly (buffer is not defined)
+import React, { useState, useEffect } from 'react';
+import cx from 'classnames';
+import { createUseStyles } from 'react-jss';
+import Color from 'color';
+import { useAuth } from  'civil-client';
+
+function Login(props, ref) {
+    
+    const { className, style, onDone = () => {}, startTab = 'login' } = props
```

Comment:
this component needs to also accept ...otherProps and then expand them into the top most div.  That way, a parent can pass down something that we havn't thought of yet, maybe onFocus or something.